src/Views/Players/PlayerDetails.js

```diff
@@ -0,0 +1,40 @@
+import React, { useEffect, useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useParams } from 'react-router-dom';
+import { fetchPlayerById } from '../../services/fetchPlayers';
+
+
+export default function PlayerDetails() {
+  const [players, setPlayers] = useState({ teams: [] });
```

Comment:
Nice job preventing the race condition of data loading versus mounting. We can also use loading state here to prevent the issue as well.